src/users/model.js

```diff
@@ -0,0 +1,23 @@
+const {DataTypes} = require("sequelize")
+const connection = require("../db/connection")
+
+const User = connection.define("User", {
+    username: {
+        type: DataTypes.STRING,
+        allowNull: false,
+        unique: true,
+    },
+    email: {
```

Comment:
Email address is not required for registration within this week's project. (backend/user/register.txt).

I realise I made the same mistake in the designs on Figma, which didn't help.